tests/main/security-device-cgroups/task.yaml

```diff
@@ -77,50 +80,62 @@ execute: |
         exit 0
     fi
 
-    echo "Given a snap is installed"
-    "$TESTSTOOLS"/snaps-state install-local test-snapd-sh
+    if [ "$SPREAD_REBOOT" = 0 ]; then
+        echo "Given a snap is installed"
+        "$TESTSTOOLS"/snaps-state install-local test-snapd-sh
 
-    echo "Then the device is not assigned to that snap"
-    udevadm info "$UDEVADM_PATH" | not MATCH "E: TAGS=.*snap_test-snapd-sh_sh"
+        echo "Then the device is not assigned to that snap"
+        udevadm info "$UDEVADM_PATH" | not MATCH "E: TAGS=.*snap_test-snapd-sh_sh"
+        udevadm info "$UDEVADM_PATH" | not MATCH "E: CURRENT_TAGS=.*snap_test-snapd-sh_sh"
```

Comment:
This will only work with systemd 247+, released in Nov 2020, so Ubuntu 21.04 and later

Reply:
Nice, thanks for that info, I already added a check for the system version